### ISSUE TITLE
Resuming training from particular epoch

### ISSUE THREAD

**Issue by dikshaaswal** [david8862/keras-YOLOv3-model-set#91]
Hi,
Is it possible to resume training from a particular epoch? As I am using google colab and it is not possible to train whole model at one go, so is it possible to resume if yes then guide please.
Thanks.

**Comment by david8862** [comment]
@dikshaaswal by default the training process will save .h5 checkpoint at "logs/000" every epoch, when val_loss has improvement. You can use "--weights_path" option in train.py to restore the needed one. "--init_epoch" could be used to specify last epoch number.

**Comment by mvanlierBCG** [comment]
@david8862 this only loads only the (trained) model weights, but not the optimizer state. Is there an option to load the optimizer state? Or do you have any suggestions? Since this can be very important.

**Comment by david8862** [comment]
@mvanlierBCG yes that's one limit in current implementation. I'm also seeking good solution for it...
